server/internal/milvus/milvus.go

```diff
@@ -102,10 +134,74 @@ func (s *S) DeleteVectorStore(ctx context.Context, name string) error {
 }
 
 // InsertDocuments inserts documents into a collection in milvus.
-func (s *S) InsertDocuments(ctx context.Context, name string, vectors [][]float32) error {
+func (s *S) InsertDocuments(ctx context.Context, name string, files, texts []string, vectors [][]float32) error {
 	vectorCol := entity.NewColumnFloatVector(vectorColName, len(vectors[0]), vectors)
-	if _, err := s.client.Insert(ctx, name, "" /* partitionName */, vectorCol); err != nil {
+	fileCol := entity.NewColumnVarChar(fileIDColName, files)
+	textCol := entity.NewColumnVarChar(textColName, texts)
+	if _, err := s.client.Insert(ctx, name, "" /* partitionName */, vectorCol, fileCol, textCol); err != nil {
 		return err
 	}
 	return nil
 }
+
+// DeleteDocuments deletes documents from a collection in milvus by fileID.
+func (s *S) DeleteDocuments(ctx context.Context, collectionName, fileID string) error {
+	if err := s.client.LoadCollection(ctx, collectionName, false); err != nil {
+		return fmt.Errorf("load collection: %s", err)
+	}
+	defer func() {
+		if err := s.client.ReleaseCollection(ctx, collectionName); err != nil {
+			log.Printf("release collection: %s\n", err)
+		}
+	}()
+
+	expr := fmt.Sprintf("%s like \"%s\"", fileIDColName, fileID)
```

Comment:
Question. Do we need to `like` here? Wondering if we can just use `=`.

Reply:
It seems `=` is for numbers only, e.g. int, float. the only operator for string type is `like`